gitlab/tox.ini

```diff
@@ -18,9 +18,6 @@ platform = linux|darwin|win32
 passenv =
     DOCKER*
     COMPOSE*
-setenv =
-    12: GITLAB_VERSION=12.10.6-ce.0
-    bench: GITLAB_VERSION=12.10.6-ce.0
```

Comment:
IMHO we should test against pinned version:
- to have more stable/predictable CI
- to keep testing against major old versions to catch regressions

Optionally, we can test against latest to catch new changes. Testing against latest has the downside of breaking CI.

cc @hithwen I think we discussed about it some time ago.

Reply:
The tradeoff here is "stable CI" vs "be alerted quickly when a backward incompatible change is merged upstream".

I think I agree with the comment for gitlab though, a prometheus based integration is not going to "break"; worse case scenario is that a new metric is added/ a metric is removed and breaks our CI for something minor.

Updating